### ISSUE TITLE
[Request] Dark themed apps like Yaru Dark

### ISSUE THREAD

**Issue by insanitywanted** [vinceliuice/WhiteSur-gtk-theme#239]
Hi Devs!

Deafult Yaru dark theme makes some app like pick, timeshift, gparted etc. dark mode. Can you please add similar feature in whitesur?

App Name is PICK (Color Picker)

**Whitesur ->** 
![issue3](https://user-images.githubusercontent.com/47140829/118265910-7cd16c80-b4d7-11eb-8ef4-2a538b65af23.png)

**Yaru Dark ->**
![issue3-1](https://user-images.githubusercontent.com/47140829/118265931-822eb700-b4d7-11eb-9778-49100769bc83.png)

**Comment by vinceliuice** [comment]
Run `./tweaks.sh -s`

**Comment by vinceliuice** [comment]
You need install this theme snap version

**Comment by vinceliuice** [comment]
https://snapcraft.io/whitesur-gtk-theme

**Comment by insanitywanted** [comment]
I installed the snap version right now. I ran the command `./tweaks.sh -s` in the terminal. 

![issue5](https://user-images.githubusercontent.com/47140829/118267830-3d584f80-b4da-11eb-9170-2e8e050aae99.png)

I am really sorry for asking these dumb question but I am kind of new to this. Thanks!

**Comment by rivanfebrian123** [comment]
> Is there any way to undo this or these changes are permanent?

Yes, it is. You can run `./tweaks.sh -s -r`. **`-r`** means for "revert"

**Comment by CyberAzure** [comment]
Just a friendly tip:

Try using Flatpak instead of Snap, Flatpak has less glitches and less inconsistency regarding custom themes.
You still have to connect WhiteSur to Flatpak, but it's easier + looks generally better.
But using official .deb and .tar.gz files are ALWAYS better.

**Comment by insanitywanted** [comment]
> Just a friendly tip:
> 
> Try using Flatpak instead of Snap, Flatpak has less glitches and less inconsistency regarding custom themes.
> You still have to connect WhiteSur to Flatpak, but it's easier + looks generally better.
> But using official .deb and .tar.gz files are ALWAYS better.

Can you link the flatpak version?

**Comment by CyberAzure** [comment]
> Can you link the flatpak version?

I've found [this](https://flathub.org/apps/details/nl.hjdskes.gcolor3) app, not exactly the same app, but has same functionality apparently, thus, less glitchy with custom themes.
There's also [this article](https://www.ubuntupit.com/best-color-picker-tools-for-linux/) showing lots of color picker apps for LiNUX in general.

And about PICK, you can [build it from scratch](https://github.com/stuartlangridge/ColourPicker) and avoid using Snap.
